i18n/zh-cn/docusaurus-plugin-content-docs/current/user_guide/workloads/run_spark.md

```diff
@@ -104,28 +108,39 @@ EOF
 kubectl proxy
 ```
 
-运行一个简单的 SparkPi 作业（这假设Spark二进制文件已安装到 `/usr/local` 目录）。
+[dockerhub](https://hub.docker.com/r/apache/spark/tags)中有不同spark版本的官方镜像
+运行一个简单的 SparkPi 作业，假设 Spark 二进制文件本地安装在 `/usr/local` 目录中。
 ```shell script
-export SPARK_HOME=/usr/local/spark-2.4.4-bin-hadoop2.7/
+export SPARK_HOME=/usr/local/spark/
 ${SPARK_HOME}/bin/spark-submit --master k8s://http://localhost:8001 --deploy-mode cluster --name spark-pi \
    --master k8s://http://localhost:8001 --deploy-mode cluster --name spark-pi \
    --class org.apache.spark.examples.SparkPi \
    --conf spark.executor.instances=1 \
    --conf spark.kubernetes.namespace=spark-test \
    --conf spark.kubernetes.executor.request.cores=1 \
-   --conf spark.kubernetes.container.image=apache/yunikorn:spark-2.4.4 \
+   --conf spark.kubernetes.container.image=docker.io/spark:v3.3.0 \
```

Comment:
docker.io/spark:v3.3.0 should be docker.io/apache/spark:v3.3.0

Reply:
updated